plugins/uefi-capsule/fu-self-test.c

```diff
@@ -258,80 +262,395 @@ fu_uefi_cod_device_func(void)
 			FU_UEFI_CAPSULE_DEVICE_STATUS_ERROR_PWR_EVT_BATT);
 }
 
+static FuVolume *
+fu_uefi_plugin_fake_esp_new(void)
+{
+	gboolean ret;
+	g_autoptr(FuVolume) esp = fu_volume_new_from_mount_path("/tmp/esp");
```

Comment:
rather than hardcode `/tmp/esp`, can we dynamically allocate and create a directory?  Otherwise we could have parallelism problems in the future for tests.

Reply:
Good idea, added.